package.json

```diff
@@ -13,7 +13,7 @@
     "e2e": "ng e2e"
   },
   "peerDependencies": {
-    "carbon-components": "10.x"
+    "@carbon/styles": "1.x"
```

Comment:
This should be a dependency, not a peer dependency (it isn't provided at runtime).